awsconfigs/common/istio-ingress/overlays/api/params.env

```diff
@@ -0,0 +1,6 @@
+# ARN of an ACM certificate to configure in ALB's HTTPS listener rule
+certArn=
+# HttpHeaderName for the HTTP header condition
+httpHeaderName=x-api-key
+# Values for the HTTP header condition
+httpHeaderValues=["token1", "token2"]
```

Comment:
Do you think we could be more explicit here and state that having 2 token values is not needed but whatever is put here will be required by the client side making the call.